pkg/runner/runner.go

```diff
@@ -32,19 +32,38 @@ type K6Runner struct {
 	Params Params
 }
 
+const K6_CLOUD = "cloud"
+const K6_RUN = "run"
+const K6_SCRIPT = "script"
+
 func (r *K6Runner) Run(execution testkube.Execution) (result testkube.ExecutionResult, err error) {
 	// check that the datadir exists
 	_, err = os.Stat(r.Params.Datadir)
 	if errors.Is(err, os.ErrNotExist) {
 		return result, err
 	}
 
-	args := []string{"run"}
+	args := []string{}
+
+	k6Subtype := strings.Split(execution.TestType, "/")[1]
```

Comment:
Looks good, but I would check, that slce has at least 2 elements to avoid out of range panic

Reply:
Good point. Improved. Done.